src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/ETInstallation.groovy

```diff
@@ -89,7 +89,8 @@ class ETInstallation extends ToolInstallation implements
 
         if (!executables.contains(exeName)) {
             throw new IllegalArgumentException("Tool executable path of '${getName()}': " +
-                    "'${home}' does not contain a tracetronic tool!")
+                    "'${home}' does not contain a tracetronic tool! " +
+                    "Please ensure the path is a full path including the executable file, not a directory.")
```

Comment:
"Please ensure the path is a full path including the executable file extension, not a directory."